Dockerfile

```diff
@@ -1,14 +1,17 @@
 ARG OS_VER
 FROM --platform=linux/amd64 nrel/openstudio:$OS_VER as buildstockbatch
 ARG CLOUD_PLATFORM=aws
-
-RUN sudo apt update && sudo apt install -y python3-pip
-RUN sudo -H pip install --upgrade pip
+ENV DEBIAN_FRONTEND=noninteractive
 COPY . /buildstock-batch/
-RUN python3 -m pip install "/buildstock-batch[${CLOUD_PLATFORM}]"
+
+RUN echo "hello"
+RUN curl -k -LsSf https://astral.sh/uv/install.sh | sh
+ENV PATH="/root/.cargo/bin:$PATH"
+RUN uv python install 3.11
+RUN uv venv --python 3.11 && uv pip install "/buildstock-batch[${CLOUD_PLATFORM}]"
 
 # Base plus custom gems
-FROM buildstockbatch as buildstockbatch-custom-gems
+FROM --platform=linux/amd64 buildstockbatch as buildstockbatch-custom-gems
```

Comment:
Ah, was it trying to rebuild that base image on aarch64? Ugh, so annoying.

Reply:
Yeah, I was getting warnings in my machine.